visualization_scripts/activate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env/bash
+
+#conda init bash
+echo "starting script"
+eval "$(conda shell.bash hook)"
+conda activate py2cyto
+
+echo "check enviornment"
+if [ $? -ne 0 ]; then
+	echo "creating py2cyto environment"
+	conda env create -f environment.yml
```

Comment:
When I ran this in Git Bash on a Windows machine, there was a prompt for administrator privileges at the end.  That usually doesn't happen when creating conda environments to my recollection.  Is that expected?

Maybe that was because of the `prefix` line in the `.yml` file? https://stackoverflow.com/questions/37926940/how-to-specify-new-environment-location-for-conda-create

Reply:
The environment also didn't install correctly.  I believe it stalled before installing the packages from pip.